authors/apps/authentication/tests/base_test.py

```diff
@@ -0,0 +1,135 @@
+from django.urls import reverse
+from rest_framework.test import APITestCase
+
+
+class BaseTest(APITestCase):
+    'class that initiates all the data required in running other tests'
```

Comment:
Follow Pep-8 standards in doc-strings, start with capital, end with period and in imperative mood. Check: https://blog.dolphm.com/pep257-good-python-docstrings-by-example/

Reply:
Ok, working on it